frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -55,16 +54,18 @@ export default class PublicGroupHero extends Component {
         <div className='container relative center'>
           <LoginTopBar loginRedirectTo={ `/${ group.slug }` } />
           <div className='PublicGroupHero-content'>
-            <UserPhoto
-              editable={canEditGroup}
-              onChange={logo => {
-                if (logo !== group.logo)
-                  return appendEditGroupForm({logo})
-              }}
-              user={{ avatar: groupForm.attributes.logo || group.logo }}
-              className='PublicGroupHero-logo mb3 bg-contain'
-              presets={[]}
-              {...this.props} />
+            {group.logo &&
```

Comment:
Mmmm... It might. The problem is that if I don't do that, the child component doesn't update once a value is fetched from the API (after we login). One way we could solve it is by returning a default logo from the API. What do you think?